app/controllers/v0/sessions_controller.rb

```diff
@@ -33,6 +33,9 @@ def saml_callback
       if @saml_response.is_valid? && persist_session_and_user
         async_create_evss_account(@current_user)
         redirect_to SAML_CONFIG['relay'] + '?token=' + @session.token
+
+        obscure_token = Session.obscure_token(@session.token)
+        Rails.logger.info("Logged in user with id #{@session.uuid}, token #{obscure_token}")
```

Comment:
Why log the `session.uuid` at all?  The `uuid` in 
```ruby
config.log_tags = [:uuid,
```
is not the same as `@session.uuid` - correct?  I'm a little hazy, but the latter comes from `request.uuid` which I _think_ comes from the revproxy adding it's own header to the request (sometimes named `X-Request-Id` but I dont think it has to be)